styles/style.css

```diff
@@ -0,0 +1,9 @@
+*{
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+    scroll-behavior: smooth;
+}
```

Comment:
Using the universal selector (*) to select all elements in CSS can cause unintended styling of elements not intended to be styled. To avoid this, it's recommended to use more specific selectors such as class, ID, or element selectors. using body instead of * would be a better option

Reply:
ok